aws/paypal-ipn-cloudformation.yml

```diff
@@ -0,0 +1,352 @@
+# paypal-ipn-cloudformation.json
+
+# receive ipn message to ipn-receiver, post to sns and return 200 ok (else 5xx no sns)
+# sns queue is ipn-topic
+# sns triggers ipn-verifier which writes to sqs on VERIFIED
+# all of the above write structured items to log .. to s3 .. for query
+
+AWSTemplateFormatVersion: "2010-09-09"
+Description: "Cloud Formation Stack for IPN interfacing; internal state on SQS unverified-ipn{-prod/-test} then verified messages delivered on SQS verified-orders{-prod/-test}"
+Parameters:
+  DeliveryQueueName:
+    Description: Delivery Queue
+    Type: String
+    Default: verified-ipn-test
+  TimeoutQueueName:
+    Description: Delivery Timeout Queue
+    Type: String
+    Default: timeout-ipn-test
+  ErrorQueueName:
+    Description: Delivery Error Queue
+    Type: String
+    Default: errors-ipn-test
+  VerifyURL:
+    Description: PaypalMessageVerificationURL
+    Type: String
+    Default: https://ipnpb.sandbox.paypal.com/cgi-bin/webscr   # for Sandbox IPNs
+    #    Default: https://ipnpb.paypal.com/cgi-bin/webscr   # for live IPNs
```

Comment:
Maybe add a comment to the readme about switching this?

Reply:
done.  added a section describing each parameter.